CHANGELOG.md

```diff
@@ -1,3 +1,393 @@
+  ```
+
+  > **NOTE:** If the update process gets interrupted, rerun above command.
+
+  > **NOTE:** If you are running self-hosted kubelet then append the above command with flag `--upgrade-kubelets`.
+
+  The update process typically takes about 10 minutes.
+  After the update, running `lokoctl health` should result in an output similar to the following:
+
+  ```bash
+  Node                     Ready    Reason          Message
+
+  lokomotive-controller-0  True     KubeletReady    kubelet is posting ready status
+  lokomotive-1-worker-0    True     KubeletReady    kubelet is posting ready status
+  lokomotive-1-worker-1    True     KubeletReady    kubelet is posting ready status
+  lokomotive-1-worker-2    True     KubeletReady    kubelet is posting ready status
+  Name      Status    Message              Error
+
+  etcd-0    True      {"health":"true"}
+  ```
+
+9. Update the bootstrap components: kubelet and etcd.
+
+  This script shipped with release tarball will update all the nodes to run the latest kubelet and etcd.
+
+  > **NOTE:** Please edit this script to disable updating certain nodes. Modify `update_other_nodes` function as required.
+
+  ```bash
+  UPDATE_BOOTSTRAP_COMPONENTS=true
+  ./lokomotive-0.6.0/scripts/update/0.5-0.6/update.sh $UPDATE_BOOTSTRAP_COMPONENTS
+  ```
+
+10. If you're using the self-hosted kubelet, reload its config.
+
+  > **NOTE:** If you're unsure you can run the command as it's harmless if you're not using the self-hosted kubelet.
+
+  ```bash
+  kubectl -n kube-system rollout restart ds kubelet
+  ```
+
+11. Re-register ingress nodes
+
+  > **NOTE:** This is an optional step and applicable only on Equinix Metal (Packet). This step ensures that the node becomes relisient to someone manually disabling the BGP from the Equinix Metal console.
+
+  > **NOTE:** SSH into a given node in a separate console upfront, before starting to follow steps.
+
+  This step ensures that you don't see any abrupt changes. Any workloads running on this node are evicted and scheduled to other nodes. The node is marked as unschedulable after running this command.
+
+  ```bash
+  export nodename=""
+  kubectl drain --ignore-daemonsets $nodename
+  ```
+
+  Delete the node object:
+
+  ```bash
+  kubectl delete node $nodename
+  ```
+
+  SSH into the node:
+
+  ```bash
+  sudo systemctl restart kubelet
+  ```
+
+  Make the node schedulable:
+
+  ```bash
+  while ! kubectl uncordon $nodename; do sleep 1; done
+  ```
+
+#### Update Docker log settings
+
+We've added log rotation to the Docker daemon running on cluster nodes. However, this only takes effect in new nodes. For this to apply to existing cluster nodes, you need to manually configure each node.
+
+- Drain the node.
+
+  This step ensures that you don't see any abrupt changes. Any workloads running on this node are evicted and scheduled to other nodes. The node is marked as unschedulable after running this command.
+
+  ```bash
+  kubectl drain --ignore-daemonsets <node name>
```

Comment:
I don't like that we ask user to drain all nodes twice. I think we could merge those instructions somehow.

Reply:
Yeah, I don't like either. But these are two unrelated tasks and I don't want to confound user by giving so much information at once.

Reply:
Or make them feel like they need to do all the steps at once or make them mix things up.

Reply:
Maybe at least we can add a cross-reference between tasks, so one can perform both at the same time?

Reply:
This is moved to a new doc.